### PR TITLE
Select2 custom ordering option

### PR DESCRIPTION
**Pull request by risinglf**
Hi!

I've created this PR with this changes:

- add **order_by** param to Ajax Filter and Ajax Input to choose the attribute used to order the results. This is very useful if you choose a custom [rasacker](https://github.com/activerecord-hackery/ransack/wiki/using-ransackers) (eg `full_name`) as the search field (that field is not mapped as a real attribute in the DB so the default order gives an error during the query)

I hope this can help you and this PR can be merged.
Thanks

### CODE REVIEW

File: spec/features/select_2_spec.rb [hunk 1]
```diff
@@ -389,5 +389,26 @@ def create_items
...
     end
+
+    context "with different order" do
+      before do
+        register_page(Category) {}
+
+        register_form(Invoice, false) do |f|
+          f.input :category_id, as: :search_select,
+                  url: admin_categories_path,
+                  order_by: :name_desc
```
Comment (monkeyci) [inline spec/features/select_2_spec.rb]: Align the elements of a hash literal if they span more than one line.

File: spec/features/select_2_spec.rb [hunk 2]
```diff
@@ -389,5 +389,26 @@ def create_items
...
       end
     end
+
+    context "with different order" do
+      before do
+        register_page(Category) {}
+
+        register_form(Invoice, false) do |f|
+          f.input :category_id, as: :search_select,
+                  url: admin_categories_path,
```
Comment (monkeyci) [inline spec/features/select_2_spec.rb]: Align the elements of a hash literal if they span more than one line.

File: spec/features/select_2_spec.rb [hunk 3]
```diff
@@ -389,5 +389,27 @@ def create_items
...
       end
     end
+
+    context "with different order" do
+      before do
+        register_page(Category) {}
+
+        register_form(Invoice, false) do |f|
+          f.input :category_id,
+                  as: :search_select,
```
Comment (monkeyci) [inline spec/features/select_2_spec.rb]: Use one level of indentation for parameters following the first line of a multi-line method call.

**Comment by emilioeduardob** [comment]
Great PR @risinglf ! thanks

**Pull request merged**